example/static/script.js

```diff
@@ -7,8 +7,20 @@ var zjb = new Zjb();
 
 (function() {
 	WebAssembly.instantiateStreaming(fetch("example.wasm"), {env: env, zjb: zjb.imports}).then(function (results) {
-		zjb.instance = results.instance;
+		zjb.setInstance(results.instance);
```

Comment:
This change needs to be made with the simple example too.  Then build the simple example and update the generated code example in the readme.